udf/aggregate_map_first.sql

```diff
@@ -35,3 +39,19 @@ CREATE OR REPLACE FUNCTION udf.aggregate_map_first(maps ANY TYPE) AS (
     ) AS key_value
   )
 );
+
+-- Test
+SELECT
+  assert_array_equals(
+    [STRUCT('k1' AS key, 'v1-1' AS value), STRUCT('k2' AS key, 'v2-1' AS value)],
```

Comment:
Choosing values that are less cryptic and look more distinctive (e.g. a/b/c) would make this slightly easier to read.